core/invoker/src/main/resources/application.conf

```diff
@@ -107,6 +107,14 @@ whisk {
     #  "POD_UID": "metadata.uid"
     #}
 
+    #if missing, the pod will be created without ephermal disk request/limit
+    #if specified, the pod will be created with ephemeral-storage request+limit set
+    #See: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#local-ephemeral-storage
+    #ephemeral-storage {
+    #  limit = 2 g
+    #  request = 500 m
```

Comment:
I know these are defaults, but It seems like a risk to have different limit and request, if exceeding the request but not the limit might cause the pod to terminate or be evicted. I can't tell from the docs if this is the case for ephemeral storage, but docs indicate it is the case e.g. for memory. Since there is currently no recovery from user-action pods that terminate due to kubernetes events despite valid configurations from OW and kubernetes point of view, it might be safer to keep these values the same. If there is no extra risk of termination, of course they can be different, I just don't know.

Reply:
That's a good point, I will limit the configuration to just a limit and set the request to the same value.